src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -227,3 +227,7 @@ export const UnfollowDescriptionAccentText = styled.span`
   font-weight: ${typography.weights.regular};
   color: ${colors.gray[50]};
 `
+
+export const NotFoundChannelContainer = styled.div`
```

Comment:
Instead creating new container, maybe we could apply these styles to `EmptyFallback` itself - like:
```tsx
const StyledEmptyFallback = styled(EmptyFallback)` ...`
```
But firstly we would have to pass `className` prop to EmptyFallback if I'm not mistaken

Reply:
Also, let's use a regular margin instead of transform

Reply:
Since not gonna be using transform did it with a flex container that's the cleanest

Did add the className to EmptyFallback anyway because why not